### ISSUE TITLE
Font size of UI items in the app (not the editor/preview)

### ISSUE THREAD

**Issue by aldanor** [HappenApps/Quiver#988]
On a macbook, everything looks great... 

... but on a 27'' screen, the entire UI is extremely tiny. 

It would be great to have an option to scale the UI up; even if it's just a single "Larger UI fonts" checkbox option to scale it up ~125%, like many other apps do (e.g. PyCharm).

**Comment by matchavez** [comment]
Actually, just exposing the application's "style" under the Style section within Settings would be fine.

**Comment by aldanor** [comment]
That would be fine too, of course (I was just thinking that a simple scale option is simpler/faster to implement than exposing stylesheets for the entire thing).

@ylian would definitely know better :) Really hoping that this is possible, one way or another.

**Comment by matchavez** [comment]
Agreed. At this point, it's more about just getting something workable in, rather than waiting for pretty.
